api/overview/azure/data-lake-store.md

```diff
@@ -65,19 +37,35 @@ dotnet add package Microsoft.Azure.Management.DataLake.Store
 
 ### Example
 
---Example overview--
+This example authenticates to an analytics account and store and creates the clients necessary for management.
 
 ```csharp
-/* Code goes here */
+// Setup authentication 
+var auth = new AdlClient.Authentication("microsoft.onmicrosoft.com"); // change this to YOUR tenant
+auth.Authenticate();
+
+// Collect info about the Azure resources needed for this connection
+string subid = "<Azure Subscription Id>";
+string rg = "<Azure Resource Group>";
+string adla_name = "<Analytics Account User Name>";
+string adls_name = "<Store Account User Name>";
+
+// Identify the accounts
+var adla_account = new AdlClient.Models.AnalyticsAccountRef(subid, rg, adla_name);
+var adls_account = new AdlClient.Models.StoreAccountRef(subid, rg, adls_name);
+
+// Create the clients
+var az = new AdlClient.AzureClient(auth);
+var adla = new AdlClient.AnalyticsClient(auth, adla_account);
+var adls = new AdlClient.StoreClient(auth, adls_account);
 ```
 
 > [!div class="nextstepaction"]
 > [Explore the management APIs](/dotnet/api/overview/azure/datalakestore/management)
 
-
 ## Samples
 
---Samples list here--
+* [Azure Data Lake .NET Client Example](https://azure.microsoft.com/en-us/resources/samples/data-lake-dotnet-client/)
 
 [PackageManager]: https://docs.microsoft.com/nuget/tools/package-manager-console
 [DotNetCLI]: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package

```

Comment:
Let's add the following link to all of the Service pages we are editing to get to the samples page too.

```
Explore more [sample .NET code](https://azure.microsoft.com/resources/samples/?platform=dotnet) you can use in your apps.
```

Put it after the bullet list of the samples.